sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs

```diff
@@ -253,10 +254,11 @@ public virtual RecognizeCustomFormsOperation StartRecognizeCustomForms(string mo
         [ForwardsClientCalls]
         public virtual RecognizeCustomFormsOperation StartRecognizeCustomFormsFromUri(string modelId, Uri formFileUri, RecognizeOptions recognizeOptions = default, CancellationToken cancellationToken = default)
         {
-            throw new NotImplementedException();
-            //SourcePath_internal sourcePath = new SourcePath_internal() { Source = formFileUri.ToString() };
-            //ResponseWithHeaders<AnalyzeWithCustomModelHeaders> response = ServiceClient.RestClient.AnalyzeWithCustomModel(new Guid(modelId), includeTextDetails: includeTextElements, sourcePath, cancellationToken);
-            //return new RecognizeCustomFormsOperation(ServiceClient, modelId, response.Headers.OperationLocation);
+            recognizeOptions ??= new RecognizeOptions();
+
+            SourcePath_internal sourcePath = new SourcePath_internal() { Source = formFileUri.ToString() };
```

Comment:
NIT: we could also do:
`var sourcePath = new SourcePath_internal(formFileUri.ToString());`